src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,18 @@
-export const TodoList = () => {};
+import { Todo } from '../../types/todo';
+import { TodoInfo } from '../TodoInfo';
+
+interface TodoListInt {
+  todos: Todo[]
+}
+
+export const TodoList = (props: TodoListInt) => {
+  const { todos } = props;
+
+  return (
+    <section className="TodoList">
+      {todos.map((todo: Todo) => (
```

Comment:
Consider adding a `key` prop to the `TodoInfo` component to improve performance. 🚀